hazelcast/src/main/java/com/hazelcast/monitor/impl/EmptyLocalReplicatedMapStats.java

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.monitor.impl;
+
+import com.hazelcast.config.ReplicatedMapConfig;
+import com.hazelcast.internal.json.JsonObject;
+import com.hazelcast.monitor.LocalIndexStats;
+import com.hazelcast.monitor.LocalReplicatedMapStats;
+import com.hazelcast.monitor.NearCacheStats;
+
+import java.util.Map;
+
+/**
+ * Implementation of {@link LocalReplicatedMapStats} with empty and immutable inner state. This is used when the statistic is
```

Comment:
so minor: better to have column width around 80

Reply:
Fixed.